tools/system_libs.py

```diff
@@ -1087,6 +1087,14 @@ def can_use(self):
     return super().can_use() and settings.USE_PTHREADS
 
 
+class libformatexception(Library):
+  name = 'libformatexception'
+  src_dir = 'system/lib'
+  src_files = ['format_exception.cpp']
```

Comment:
Any reason not to just include this in libcxxabi?  It shouldn't have any code unless its actually used.

Reply:
Sure, I can move it there.